src/main/java/pl/scoutCamp/business/RankingService.java

```diff
@@ -15,21 +17,20 @@
 import java.util.Comparator;
 import java.util.List;
 
+import static pl.scoutCamp.api.controller.enums.GivenCategory.*;
+
 @Slf4j
 @Service
 @AllArgsConstructor
 public class RankingService {
 
     private final TeamDAO teamDAO;
     private final TeamCategorizationSheetDAO teamCategorizationSheetDAO;
-    public static final String POINTS = "points";
-    public static final String NAME = "name";
-    public static final String CATEGORY = "category";
 
     public Page<Ranking> createRankingList(
             String period,
```

Comment:
I find it hard to replace String period to enum here, since enums are static. Every year we would must to add new value with period in this enum. It's impossible to create them dynamicly based on database.